backend/fms_core/serializers.py

```diff
@@ -55,6 +58,29 @@ class Meta:
         model = SampleKind
         fields = "__all__"
 
+class ProtocolSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Protocol
+        fields = "__all__"
+
+class ProcessSampleSerializer(serializers.ModelSerializer):
+    protocol = serializers.IntegerField(read_only=True, source="process.protocol.id")
+    child_sample = serializers.IntegerField(read_only=True)
+
+    class Meta:
+      model = ProcessSample
+      fields = "__all__"
+      extra_fields = ('protocol', 'child_sample')
+
+class ProcessSampleExportSerializer(serializers.ModelSerializer):
+    protocol_name = serializers.CharField(read_only=True, source="process.protocol.name")
+    child_sample_name = serializers.CharField(read_only=True)
+    source_sample_name = serializers.CharField(read_only=True)
+
+    class Meta:
+      model = ProcessSample
+      fields = ('id', 'protocol_name', 'source_sample_name', 'child_sample_name', 'volume_used', 'execution_date', 'comment')
+      extra_fields = ('protocol_name', 'source_sample_name', 'child_sample_name')
```

Comment:
You dont need to have the fields declared in extra_fields too, it is already declared in fields

Reply:
Removed extra_fields